recipes/cylc-flow/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "cylc-flow" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 36f4f4363cb5cc7a3a613d2db6e84cc41c2825c6cd011191cc4821c5ca9c81a4
+
+build:
+  number: 0
+  skip: True  # [py<37 or not linux]
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - ansimarkup >=1.0.0
+    - colorama >=0.4
+    - cryptography
+    - graphene ==2.1.6
+    - jinja2 >=2.10.1
+    - markupsafe >=1.1
+    - metomi-isodatetime >=1!2.0.1
+    - protobuf >=3.7
+    - pyasn1
+    - python
+    - python-jose >=3.0
+    - pyzmq >=18.0
+
+test:
+  imports:
+    - cylc.flow
+  commands:
+    - cylc --version
+
+about:
+  home: https://cylc.github.io/
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: 'A workflow engine for cycling systems'
```

Comment:
Do we actually need the quotes here?

Reply:
I added these in because when I looked at various other ``meta.yaml`` files linked in from under ``conda-forge/feedstocks``, they all had single quotes here. See e.g. (picked arbitrarily from the listing [here](https://github.com/conda-forge/feedstocks/tree/master/feedstocks)) [this](https://github.com/conda-forge/alembic-feedstock/blob/a9d5f20889086ba748efc9e6449ed1fa5f25bdeb/recipe/meta.yaml), [this](https://github.com/conda-forge/advbumpversion-feedstock/blob/3df23803bdb59631a0d5214cc87cc7c7038712e0/recipe/meta.yaml) & [this](https://github.com/conda-forge/adios-feedstock/blob/e3267e19ad950d5564f538b61e270f5347942cd0/recipe/meta.yaml) recipe.

Some other examples there do not use single quotes, but I get the impression it is good practice to, for some reason? I'll look at the docs for any clues in that respect.